### PR TITLE
Enable multi-chain deployment

### PR DESCRIPTION
**Pull request by ACC01ADE**
## Describe Changes

sample multi-chain deployment

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] Code styles have been enforced
- [x] All Hardhat tests are passing

### CODE REVIEW

**Comment by alexanderattar** [comment]
We should add a test for this function. Otherwise this seems like a simple an obvious upgrade we should make

**Review by sogoiii — APPROVED** [review]
The logic does make sense.

File: contracts/HolographFactory.sol [hunk 1]
```diff
@@ -193,7 +224,7 @@ contract HolographFactory is Admin, Initializable, Holographable, HolographFacto
     DeploymentConfig memory config,
     Verification memory signature,
     address signer
-  ) external {
+  ) public {
```
Comment (alexanderattar) [inline contracts/HolographFactory.sol:227]: why the change to public visibility?

**Review by alexanderattar — APPROVED** [review]

File: contracts/HolographFactory.sol (hunk 1 above)
Comment (sogoiii) [reply contracts/HolographFactory.sol:227]: I think it was changed because its now being called from within this contract `deployHolographableContractMultiChain`.
